### PR TITLE
Remove useless system dependencies

### PR DESCRIPTION
**Pull request by Luni-4**
Binaries are contained into the `PyPI` packages

### CODE REVIEW

**Comment by rr-** [comment]
Hmm, are you 100% sure about this? I remember having to install these, which is why I put it in readme in the first place.

**Comment by Luni-4** [comment]
I have removed them from the Dockerfile and haven't had any problem. I will make a more in-depth check though. I will let you know soon.

**Comment by Luni-4** [comment]
Only `PyQt5` can be installed directly from `pip` without having to install additional binaries.

File: README.md [hunk 1]
```diff
@@ -100,5 +99,5 @@ plugin](docs/example_plugin.py))
 
 I'd like to thank all
 [contributors](https://github.com/rr-/bubblesub/graphs/contributors) for the
-help on this project.  
+help on this project.
```
Comment (rr-) [inline README.md]: This change destroys formatting.

Comment (Luni-4) [reply README.md]: Done, thanks!

**Pull request merged**